gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dinfoitem_ttt2.lua

```diff
@@ -0,0 +1,85 @@
+---
+-- @class PANEL
+-- @section DInfoItemTTT2
+
+local PANEL = {}
+
+---
+-- @ignore
+function PANEL:Init()
+	self:SetContentAlignment(5)
+
+	self:SetTall(22)
+	self:SetMouseInputEnabled(true)
+	self:SetKeyboardInputEnabled(true)
+
+	self.data = {}
+end
+
+---
+-- @param table data
+-- @realm client
+function PANEL:SetData(data)
+	self.data = data
+end
+
+---
+-- @return string
+-- @realm client
+function PANEL:GetText()
+	return self.data.text.text
```

Comment:
This is a larger refactor because the variables are named that way everywhere in the code. If this really is important to you, I can change it. At some point I switched to body and title instead of text and title, but there is still old code I interact with that I did not want to rework